### ISSUE TITLE
No Disks in report (10.13 BETA 6 - 17A344b)

### ISSUE THREAD

**Issue by marklajer** [munkireport/munkireport-php#858]
After upgrading test machines to 10.13 Beta 6 (17A344b) running APFS.

Munki is not able to see the disk anymore.

```
Server An error occurred while processing: \Disk_report_model
Server Error: No Disks in report
```

```
cat /usr/local/munki/preflight.d/cache/disk.plist
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<array>
</array>
</plist>
```

**Comment by bochoven** [comment]
disk_info is broken for APFS. Some functions need to be rewritten

**Comment by bochoven** [comment]
Fixed in MR 2.15
